### DISCUSSION TITLE
QField Sync 'The Project You Imported Does Not Appear to be an Offline Project'

### DISCUSSION THREAD

**Discussion by timjosmith** [opengisch/QField#2708] [Q&A]
When using QField Sync the import from mobile device back to QGIS on my desktop PC fails with a message saying 'The Project You Imported Does Not Appear to be an Offline Project'.  I configured the layers before loading onto my mobile device to Copy and did not select Offline Editing.

Please could anybody advise on how to resolve this issue so I can import all my data collected using Qfield on my mobile device to QGIS 3.16 Hannover using Qfield Sync?

I am experienced in QGIS and have sucessfully used Qfield several times but have not experienced this issue until now.

**Comment by gdsrg** [comment]
I'm having exactly the same problem. But as far as I can tell my collected data did transfer correctly, despite the error message. I would love to be sure that the message can be ignored, though.

**Comment by timjosmith** [comment]
Yes, my shapefile came across too but did not contain any attributes and also will not display in the map window despite being able to the layer loaded correctly in the layer panel.  Have carried out all the normal checks and tried resyncing using Qfield Sync several time but to no avail.  Wish I could get a response on the forum.  Hope you resolve your issue.   

 

Thanks

 

Tim

**Reply by gdsrg** [reply]
I used geopackage layers for collecting data.  I get the error, but the data seems OK.

When I used shape files (as 'copy' not 'offline editing') QField seemingly lets me collect data fine without error message, but the shape file is corrupted afterwards and can't be opened at all (similar to yours).
I think the data collection layer has to be set to 'offline editing' when using shape files. The packaging of the project then creates a data.gpkg file in which the collected data gets stored and then played back to your QGIS project. 

(That can create new errors though. I can't use 'offline editing' for any of my vector layers, because for some reason, as soon as there is a WMS present in the project, all my vector layers seem fine in QGIS but on my mobile device they show as missing. But that's a different matter.)

**Comment by timjosmith** [comment]
Thanks very much, that sounds plausible and makes a lot of sense as when I have used shapefiles as ‘offline editing’ there has not been a problem; I only received the error message and missing all the attributes of the shapefile when I used ‘copy’ when configuring the project.
